_topic_maps/_topic_map_osd.yml

```diff
@@ -304,6 +304,172 @@ Topics:
 - Name: Exposing the registry
   File: securing-exposing-registry
 ---
+Name: Operators
+Dir: operators
+Distros: openshift-dedicated
+Topics:
+- Name: Operators overview
+  File: index
+- Name: Understanding Operators
+  Dir: understanding
+  Topics:
+  - Name: What are Operators?
+    File: olm-what-operators-are
+  - Name: Packaging format
+    File: olm-packaging-format
+  - Name: Common terms
+    File: olm-common-terms
+  - Name: Operator Lifecycle Manager (OLM)
+    Dir: olm
+    Topics:
+    - Name: Concepts and resources
+      File: olm-understanding-olm
+    - Name: Architecture
+      File: olm-arch
+    - Name: Workflow
+      File: olm-workflow
+    - Name: Dependency resolution
+      File: olm-understanding-dependency-resolution
+    - Name: Operator groups
+      File: olm-understanding-operatorgroups
+    - Name: Multitenancy and Operator colocation
+      File: olm-colocation
+    - Name: Operator conditions
+      File: olm-operatorconditions
+    - Name: Metrics
+      File: olm-understanding-metrics
+    - Name: Webhooks
+      File: olm-webhooks
+  - Name: OperatorHub
+    File: olm-understanding-operatorhub
+  - Name: Red Hat-provided Operator catalogs
+    File: olm-rh-catalogs
+  - Name: Operators in multitenant clusters
+    File: olm-multitenancy
+  - Name: CRDs
+    Dir: crds
+    Topics:
+    - Name: Managing resources from CRDs
+      File: crd-managing-resources-from-crds
+- Name: User tasks
+  Dir: user
+  Topics:
+  - Name: Creating applications from installed Operators
+    File: olm-creating-apps-from-installed-operators
+- Name: Administrator tasks
+  Dir: admin
+  Topics:
+  - Name: Adding Operators to a cluster
+    File: olm-adding-operators-to-cluster
+  - Name: Updating installed Operators
+    File: olm-upgrading-operators
+  - Name: Deleting Operators from a cluster
+    File: olm-deleting-operators-from-cluster
+  - Name: Configuring proxy support
+    File: olm-configuring-proxy-support
+  - Name: Viewing Operator status
+    File: olm-status
+  - Name: Managing Operator conditions
+    File: olm-managing-operatorconditions
+  - Name: Managing custom catalogs
+    File: olm-managing-custom-catalogs
+  - Name: Catalog source pod scheduling
+    File: olm-cs-podsched
+# - Name: Managing platform Operators  <= Tech Preview
+#   File: olm-managing-po
+- Name: Developing Operators
+  Dir: operator_sdk
+  Topics:
+  - Name: About the Operator SDK
+    File: osdk-about
+  - Name: Installing the Operator SDK CLI
+    File: osdk-installing-cli
+  - Name: Go-based Operators
+    Dir: golang
+    Topics:
+#   - Name: Getting started
```

Comment:
Do you want to add a note about why the quickstarts are not in ROSA/OSD?

Reply:
Yeah, probably a good idea. I added comments to the OSD and ROSA topic maps to note why the quick starts are excluded.